### PR TITLE
Replaces local dependency with pods dependency

### PR DESCRIPTION
**Pull request by vegaro**
Cordova handles the pod installation automatically and this will simplify iOS installation and remove the necessity to `strip_frameworks.sh`.

Existing projects might need to re-add the iOS platform.

### CODE REVIEW

**Comment by jasonandress** [comment]
I use PhoneGap Build and with this branch I received the following error.

In file included from /project/<......>/Plugins/cordova-plugin-purchases/RCPurchaserInfo+CDVPurchasesPlugin.m:9:
/project/<.....>/Plugins/cordova-plugin-purchases/RCPurchaserInfo+CDVPurchasesPlugin.h:9:9: fatal error: 'Purchases/Purchases.h' file not found
#import <Purchases/Purchases.h>
        ^~~~~~~~~~~~~~~~~~~~~~~
1 error generated.

File: plugin.xml [hunk 1]
```diff
@@ -33,6 +33,13 @@
         <source-file src="src/ios/RCPurchaserInfo+CDVPurchasesPlugin.m" />
         <header-file src="src/ios/SKProduct+CDVPurchasesPlugin.h" />
         <source-file src="src/ios/SKProduct+CDVPurchasesPlugin.m" />
-        <framework src="src/ios/Purchases.framework" embed="true" custom="true" />
+        <podspec>
```
Comment (jasonandress) [inline plugin.xml]: I believe the problem is <podspec> is only supported by cordova-ios 5.0.0 Phonegap build latest supported is 4.5.5 Can you update it to the old way? Or I can fork this and do it myself.

<framework src="Purchases" spec="~> 2.3.0" />

**Comment by jasonandress** [comment]
Now I get a basic Pod error
Repository "https://github.com/revenuecat/cordova-plugin-purchases.git" checked out to git ref "pods" at "45eb1d8".
Installing "cordova-plugin-purchases" at "0.1.2" for ios
Fetching plugin "cordova-annotated-plugin-android" via npm
Installing "cordova-annotated-plugin-android" at "1.0.4" for ios
Failed to install 'cordova-plugin-purchases': Error: pod: Command failed with exit code 1
    at ChildProcess.whenDone (/private/project/cordova/node_modules/cordova-common/src/superspawn.js:169:23)
    at ChildProcess.emit (events.js:182:13)
    at maybeClose (internal/child_process.js:962:16)
    at Socket.stream.socket.on (internal/child_process.js:381:11)
    at Socket.emit (events.js:182:13)
    at Pipe._handle.close (net.js:611:12)
pod: Command failed with exit code 1

**Comment by vegaro** [comment]
You might need to specify the iOS deployment target to 9.0. According to PhoneGap's documentation (http://docs.phonegap.com/phonegap-build/configuring/preferences/#example) it should be the following:
​
​`<preference name="deployment-target" value="9.0" />​`
​
​In the config.xml

**Comment by vegaro** [comment]
Closing since this is only fully supported by `cordova-ios@5.0.0`. Previous versions don't add the deployment target to the project's Podfile, which makes it brake the installation.

**Pull request closed without merging**